themes/root/templates/RecordDriver/EDS/export-ris.phtml

```diff
@@ -0,0 +1,108 @@
+<?
+$items = $this->driver->getItems();
```

Comment:
First, I used the following documentation of RIS format http://referencemanager.com/sites/rm/files/m/direct_export_ris.pdf
following methods are a duplicate of the AbstractBase version:
getISSNs
getContainer*

These methods are unlike to the AbstraceBase implementation
getAuthors (Ebsco does not really distinguish between first, second, etc author. Sometimes they are sorted randomly (by priority?), sometimes alphabetically. 
getPubType, getFormats, the abstract implementation has far too less possible formats, at least for non-marc records. 

What's your suggestion?